src/libraries/System.Security.Cryptography.ProtectedData/src/PACKAGE.md

```diff
@@ -2,43 +2,82 @@
 
 <!-- A description of the package and where one can find more documentation -->
 
+System.Security.Cryptography.ProtectedData offers a simplified interface for utilizing Microsoft Windows DPAPI's [CryptProtectData](https://learn.microsoft.com/windows/win32/api/dpapi/nf-dpapi-cryptprotectdata) and [CryptUnprotectData](https://learn.microsoft.com/windows/win32/api/dpapi/nf-dpapi-cryptunprotectdata) functions.
 
+This library is primarily aimed at ensuring the security of critical data, like user credentials or sensitive information, either while storing or transmitting.
```

Comment:
I'd delete this paragraph entirely.

The plain truth is that it's primarily aimed at people already using it.  It's reasonable for storing sensitive data locally, not generally suitable for transmission (except for a domain user sending it to themselves on another computer; or for "give this back to me"), but since it's not cross-plat it's not something that we really push... so... it's there for people who were using ProtectedData in .NET Framework.

Reply:
Should we give explicit guidance like that? Sounds almost like "don't introduce new use, consider XYZ instead"

Reply:
Removing this statement doesn't hurt the documentation, so I'm fine with it.

There is already a note about the better cross-plat support in `System.Security.Cryptography`.